src/Examples/Advanced/CustomUnitModels/methanol_param_VLE.py

```diff
@@ -114,17 +118,23 @@ def build(self):
                                      ('H2', 'C'): 3.19,
                                      ('CH3OH', 'A'): 18.5875,
                                      ('CH3OH', 'B'): 3626.55,
-                                     ('CH3OH', 'C'): -34.29} # these Antoine coefficients assume mmHG and K
+                                     ('CH3OH', 'C'): -34.29}
 
         Cp = self.config.Cp
-        Cv = Cp - Constants.gas_constant/1000/pyunits.mol*pyunits.kmol # convert gas constant from J/mol-K to MJ/kmol-K
+        Cv = Cp - pyunits.convert(Constants.gas_constant,
```

Comment:
It isn't really necessary here, but I think it would be good practice to use the Pyomo `value()` method to evaluate this expression. In this case you get by without it as all parts of the expression are constants, but if anything was a variable (if the general sense), then you would see an exception when using this.